scripts/docker/release.Dockerfile

```diff
@@ -19,24 +19,25 @@ ENV RUST_BACKTRACE 1
 
 # install tools and dependencies
 RUN apt-get update && \
-		DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-			libssl1.1 \
-			ca-certificates \
-			curl \
-			gnupg && \
-		useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot && \
-		gpg --recv-keys --keyserver hkps://keys.mailvelope.com 9D4B2B6EB8F97156D19669A9FF0812D491B96798 && \
-		gpg --export 9D4B2B6EB8F97156D19669A9FF0812D491B96798 > /usr/share/keyrings/parity.gpg && \
-		echo 'deb [signed-by=/usr/share/keyrings/parity.gpg] https://releases.parity.io/deb release main' > /etc/apt/sources.list.d/parity.list && \
-		apt-get update && \
-		apt-get install -y --no-install-recommends polkadot=${POLKADOT_VERSION#?} && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+		libssl1.1 \
+		ca-certificates \
+		curl \
```

Comment:
Adding `curl` involves potential security risks. Since `curl` is not required for running `polkadot`, could we either:
- remove the curl dependency entirely ?
- remove `curl` once we are done with it ?

Reply:
I do not believe we depend on `curl`, it's there for debug purposes.

Reply:
happy to remove it

Reply:
let's merge it as is here, I'll create a separate issue for this.

Reply:
See https://github.com/paritytech/ci_cd/issues/197